### PR TITLE
Add support for configured Elasticsearch URL

### PR DESCRIPTION
**Pull request by mohabusama**
+ URL can be overridden in check-command to query external ES cluster

### CODE REVIEW

File: zmon_worker_monitor/builtins/plugins/elasticsearch.py [hunk 1]
```diff
@@ -33,24 +33,29 @@ class ElasticsearchFactory(IFunctionFactoryPlugin):
...
 
+        self._url = None
+
     def configure(self, conf):
         """
         Called after plugin is loaded to pass the [configuration] section in their plugin info file
         :param conf: configuration dictionary
         """
-        return
+        self._url = conf.get('url')
```
Comment (Jan-M) [inline zmon_worker_monitor/builtins/plugins/elasticsearch.py]: Are we sure this gets injected from entity, if eg entity has a url field?

```
{"type":"es-cluster", "id:"my-es-cluster-1", "url":"https://es-cluster......"}
```

Comment (mohabusama) [inline zmon_worker_monitor/builtins/plugins/elasticsearch.py]: Haven't tested this scenario. Will give it a try.

Comment (mohabusama) [inline zmon_worker_monitor/builtins/plugins/elasticsearch.py]: @Jan-M from what I see, it doesn't get injected, only the value in the config is used. Is it supposed to be auto-injected or should it be used in the check command (e.g. ``elasticsearch(entity['url'])...`` ) ?

Comment (Jan-M) [inline zmon_worker_monitor/builtins/plugins/elasticsearch.py]: It should be injects, can you compare with http() or sql()?

Comment (mohabusama) [inline zmon_worker_monitor/builtins/plugins/elasticsearch.py]: Ok. Done and tested.

precedence:

+ check command url 
+ entity url
+ config url

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**